lib/rules/no-curly-component-invocation.js

```diff
@@ -211,6 +202,14 @@ module.exports = class NoCurlyComponentInvocation extends Rule {
       },
     };
   }
+
+  findAttrParent(path) {
```

Comment:
You could extract a `findFirstParentOfType(path, type)` function into a helper file so it could be reused across rules.

Reply:
I've found a simpler way. Would be awesome if you could have another look 🙏

Reply:
LGTM.